scripts/install.sh

```diff
@@ -0,0 +1,18 @@
+#install the libraries below
+arduino-cli lib install Adafruit_Keypad@1.0.4
+arduino-cli lib install Adafruit_MAX31865@1.0.3 #Has a duplicate maybe? May need a double check
+arduino-cli lib install PID@1.2.1
+arduino-cli lib install ArduinoJson@6.14.1
+arduino-cli lib install AsyncTaskLib@1.0.0
+arduino-cli lib install Ethernet@2.0.0
+arduino-cli lib install Keypad@3.1.1
+arduino-cli lib install LiquidCrystal@1.0.7
+arduino-cli lib install Marceau@0.0.1
+arduino-cli lib install MemoryFree@latest #Special, hosted locally with no properties page
+arduino-cli lib install PID_AutoTune_v0@0.0.1 #Has no dedicated properties page
+arduino-cli lib install RTClib@1.3.3
+arduino-cli lib install SD@1.2.4
+arduino-cli lib install Time@1.6
+arduino-cli lib install TimedAction@2.0
+arduino-cli lib install TimerOne@1.1
+arduino-cli lib install TrueRandom@1.1 #Current version not certain, version written in is based on release notes
```

Comment:
Time, TimedAction, and TrueRandom also appear to not have Arduino libraries accessible from the CLI